devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -63,7 +63,7 @@
 
     private Set<File> filesIncludedInClasspath = new HashSet<>();
 
-    private String debug;
+    private String debug = System.getProperty("debug");
```

Comment:
This line can be removed. The `startDev` method is the only method that use/read the debug flag so it's better to move this code line to that method and read in into a method local variable debug. Place it right before the if statements.

Reply:
You're right, thanks! That's fixed now.